c++/src/kj/compat/tls-test.c++

```diff
@@ -424,6 +427,27 @@ struct TlsTest {
     options.trustedCertificates = kj::arrayPtr(&caCert, 1);
     return options;
   }
+
+  Promise<void> writeToServer(Own<AsyncIoStream> client) {
+    return client->write("foo\0", 4).attach(kj::mv(client));
+  }
+
+  Promise<void> readFromClient(Own<AsyncIoStream> server) {
+    auto buf = heapArray<char>(4);
+    return server->read(buf.begin(), buf.size())
+        .attach(kj::mv(server))  //
```

Comment:
Super minor, I was curious if you meant to comment about something here or if this `//` was just leftover.

Reply:
Well, no. It's actually a way to kludge clang-format into doing a nice function chaining. That said, this is an appropriate spot for a small comment explaining what we're doing.

Reply:
Unfortunately clang-format doesn't really support our style (yet -- Vitali has been submitting some changes upstream). I don't recommend relying on it for now.

E.g. the `.then()` below here indents the lambda body too much. We indent lambda bodies one stop more than the surrounding function.